subiquity/server/controllers/shutdown.py

```diff
@@ -100,8 +100,19 @@ async def copy_logs_to_target(self, context):
         if self.opts.dry_run:
             os.makedirs(target_logs, exist_ok=True)
         else:
+            cloudint_logs = [
+               cloudinit_log
+               for cloudinit_log in (
+                    "/var/log/cloud-init.log",
+                    "/var/log/cloud-init-output.log"
+               )
+               if os.path.exists(cloudinit_log)
+            ]
+            await arun_command(
+                ['cp', '-a', " ".join(cloudinit_logs), '/var/log/installer'])
```

Comment:
I'm not sure how it would happen, but this form is invalid if neither log file exists I think.

Reply:
The default value of check is False, so I guess just try to copy them over and if they don't exist, no big deal.